torch/profiler/profiler.py

```diff
@@ -98,6 +98,18 @@ class profile(object):
         The default schedule simply records all the events continuously for the
         duration of the context manager.
 
+    .. note::
+        Use ``torch.profiler.tensorboard_trace_handler`` to generate result files for tensorboard:
+
+        ``on_trace_ready=torch.profiler.tensorboard_trace_handler(dir_name)``
+
+        After profiling, result files will be output into that dir and then can use command
```

Comment:
After profiling, result files can be found in the specified directory, the user can use the command:  to see the result view in TensorBoard